aggregator/src/aggregator.rs

```diff
@@ -271,7 +274,7 @@ impl<C: Clock> Aggregator<C> {
             clock,
             cfg,
             report_writer,
-            task_aggregators: Mutex::new(HashMap::new()),
+            task_aggregators: SyncMutex::new(HashMap::new()),
```

Comment:
nit: we only use the std::sync::Mutex so we don't need to alias this type to SyncMutex.

Reply:
Not so -- the inner mutexes are Tokio's `Mutex` implementation, and they need to be since they are held across await points.

Reply:
Ah, oops!